lib/launchers/process.js

```diff
@@ -70,7 +70,7 @@ function ProcessLauncher (spawn, tempDir, timer, processKillTimeout) {
       return self._clearTempDirAndReportDone('no binary')
     }
 
-    cmd = this._normalizeCommand(cmd)
+    cmd = self._normalizeCommand(cmd)
```

Comment:
@devoto13, @johnjbarton 
What do you thinks about that ? I did something opposite here:
https://github.com/karma-runner/karma/pull/3157

Reply:
Consider this example: https://jsfiddle.net/devoto13/1ptvhgxn/

Apparently the only case, where it has incorrect `this` is when method is used by reference (i.e. `.then(myClass.myMethod)`), which is ultimately shooting yourself in the foot. So I think your (@lusarz's) way is good as long as we (and consumers) avoid this pattern.

But maybe I'm missing something. @Itee can you provide a more complete example of when this breaks?

Reply:
But note that changes from self to this can potentially be breaking to the third-party code, which uses the mentioned pattern, so should probably go into a major version.

Reply:
Thanks @devoto13 for great input - I'll analyze your example more precisely tomorrow, but I think I know what you mean.

That's true - it would be better to release major version with these changes